### PR TITLE
Misc fixes for the examples

### PR DESCRIPTION
**Pull request by epolack**
Small fixes for the doc, mostly removing useless returns.
I ran it locally and the results seem alright.
~Don't know how to fix ugly `Apply ε† = 1 - χ0 (vc + fxc)` of `dielectric.jl`, so I didn't :p~

### CODE REVIEW

File: examples/anyons.jl [hunk 1]
```diff
@@ -6,18 +6,19 @@ using DFTK
 using StaticArrays
 using Plots
 
-## Unit cell. Having one of the lattice vectors as zero means a 2D system
+# Unit cell. Having one of the lattice vectors as zero means a 2D system
```
Comment (antoine-levitt) [inline examples/anyons.jl:9]: Meh, these are technical comments that don't really merit a separate markdown section

Comment (antoine-levitt) [reply examples/anyons.jl:9]: More worryingly, the plot in https://docs.dftk.org/dev/examples/anyons/ doesn't show up anymore, @mfherbst do you know what's up with that?

**Review by antoine-levitt — COMMENTED** [review]
This is super annoying to get right, so thanks, this looks good!

File: examples/input_output.jl [hunk 2]
```diff
@@ -29,7 +29,7 @@ qe_input  = "Fe_afm.pwi"
 atoms     = load_atoms(qe_input)
 positions = load_positions(qe_input)
 lattice   = load_lattice(qe_input)
-magnetic_moments = load_magnetic_moments(qe_input)
+magnetic_moments = load_magnetic_moments(qe_input);
```
Comment (mfherbst) [inline examples/input_output.jl:32]: I think it's nice to see these
```suggestion
magnetic_moments = load_magnetic_moments(qe_input)
```

**Review by mfherbst — COMMENTED** [review]
Thanks very much @epolack!

**Pull request merged**